packages/pluggableWidgets/image-native/src/package.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <package xmlns="http://www.mendix.com/package/1.0/">
-    <clientModule name="Image" version="1.1.0" xmlns="http://www.mendix.com/clientModule/1.0/">
+    <clientModule name="Image" version="1.1.1" xmlns="http://www.mendix.com/clientModule/1.0/">
```

Comment:
should we update the package.json of the widget as well or is changing package.xml enough ?

Reply:
Yes, we should update the package.json as well